spec.html

```diff
@@ -6196,11 +6207,12 @@ <h1>CreateIntrinsics ( _realmRec_ )</h1>
       <h1>SetRealmGlobalObject ( _realmRec_, _globalObj_, _thisValue_ )</h1>
       <p>The abstract operation SetRealmGlobalObject with arguments _realmRec_, _globalObj_, and _thisValue_ performs the following steps:</p>
       <emu-alg>
+        1. Let _intrinsics_ be _realmRec_.[[Intrinsics]].
         1. If _globalObj_ is *undefined*, then
-          1. Let _intrinsics_ be _realmRec_.[[Intrinsics]].
           1. Set _globalObj_ to ObjectCreate(_intrinsics_.[[%ObjectPrototype%]]).
         1. Assert: Type(_globalObj_) is Object.
         1. If _thisValue_ is *undefined*, set _thisValue_ to _globalObj_.
+        1. Set _intrinsics_.[[%GlobalThisValue%]] to _thisValue_.
```

Comment:
Was it considered to store it directly on Realm next to `[[GlobalObject]]`? I think I need to reuse this setup to properly define `window.self` and friends and it's a bit weird to go through [[Intrinsics]]: https://github.com/whatwg/html/pull/4410.

Reply:
+1, although it doesn't observably matter, it seems like the global this value pointer should be more first-class than using the "saved intrinsics table".

Reply:
That does make sense - it seems like that would involve not having an intrinsic at all.

There's already an abstract operation (`GetThisBinding`) that can give you the value, but I'm not certain how to phrase that in the actual setup for the global data property.